### PR TITLE
report for 12.08.2024

### PR DESCRIPTION
**Pull request by hansmsc90**

### CODE REVIEW

File: orcestra_book/reports/METEOR-20240812.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+mission: ORCESTRA
+platform: METEOR
+report_id: METEOR-20240811
```
Comment (lkluft) [inline orcestra_book/reports/METEOR-20240812.md:4]: Could you please update the report id? ☺️

**Review by lkluft — CHANGES_REQUESTED** [review]
Besides the report id, the changes look fine 👌

**Comment by hansmsc90** [comment]
Changes are done :) 
Greeting from Mindelo, Lukas.

**Review by lkluft — APPROVED** [review]
Thanks for the fix! Best wishes and hopefully a soon start to you ;)

**Pull request merged**